### DISCUSSION TITLE
Save button does nothing??

### DISCUSSION THREAD

**Discussion by dax702** [typemill/typemill#477] [Q&A]
I'm testing out TypeMill and I notice that when I click the Save button in the content editor, nothing happens. And when I try to click on another element it gives me a notice "Save or cancel your changes first."  Why doesn't the Save button work? I'm using Brave browser, latest version in Windows 10 (happens with shields up or down) Any ideas?

**Comment by trendschau** [comment]
I just tested the latest version with Firefox and Chrome + Windows and Linux without errors, so I suppose it is a local issue. Maybe the writing permissions for the content folder are missing or some other server configuration? 

You can try to narrow down the error this way: 

* Go to system settings, open the developer tab and activate the checkbox for error reporting (don't forget to deactivate it after testing again).
* Open the developer tools in your browser and then reproduce the error once again. 

You should then see a response or error message in the developer tools under "network". Just click on the red in the list of network connections and the details will open in another window. You have to click on the tab "response" to see the response from the server like in the screenshot (with firefox):

![grafik](https://github.com/typemill/typemill/assets/2714547/4f2d65cf-de85-4454-bdb9-2d28ebd7d75f)
